app/controllers/orders_controller.rb

```diff
@@ -14,6 +14,15 @@ def show
     @order_items = @order.shopping_cart_items
   end
 
+  def destroy
```

Comment:
The name of this method indicates that you are deleting the entire order. Might need to be refactored (renamed). I'll let it slip for now but make sure you are on top of things and not use this method for anything else BUT removing order items.